main/client/static/tag/zenTable.tag

```diff
@@ -1,8 +1,6 @@
 <zentable class="containerV {opts.zentableClass}" style="flex-grow:1;">
   <!-- header -->
-  <div class="containerH tableHeader" ref="tableHeader" >
-    <yield from="header"/>
-  </div>
+  <div class="containerH tableHeader" ref="tableHeader" ><yield from="header"/></div>
```

Comment:
`:blank` selector does not exist, we use `:empty` but if we add a new line, it brokes the style. :cry: